tests/e2e/new_mnemonic.rs

```diff
@@ -255,8 +372,8 @@ fn test_withdrawal_credentials_execution() -> Result<(), Box<dyn std::error::Err
     cmd.arg(decryption_password);
     cmd.arg("--num_validators");
     cmd.arg(num_validators);
-    cmd.arg("--withdrawal_credentials");
-    cmd.arg(execution_withdrawal_credentials);
+    cmd.arg("--kdf");
+    cmd.arg(kdf);
```

Comment:
This should also test that generated keystore has proper encryption specified, e.g.

given generated keystore record is like
```
{
  "crypto": {
    "checksum": {
      "function": "sha256",
      "message": "b95e2bf5c5aafe0e9149f08d2e38f1daa5d20b73972433cd1b523e647249544c"
    },
    "cipher": {
      "function": "aes-128-ctr",
      "message": "bbed72f35ea11f66a44ca42c5b593574db1fb1bd763c91e64a8629899fb58ffe",
      "params": {
        "iv": "793b3201f800b017e80c35b5486ab5df"
      }
    },
    "kdf": {
      "function": "scrypt",
      "message": "",
      "params": {
        "dklen": 32,
        "n": 262144,
        "p": 1,
        "r": 8,
        "salt": "14260c0c72c4afcde2b99ccdf2e1b1257d3cb5dfdc19ddbdc70f2b36eee90b9a"
      }
    }
  },
  "description": "",
  "path": "m/12381/3600/9/0/0",
  "pubkey": "b9d798bbff9a7d7599617035c1141ef607f0b4398ad9c3753219c634397b0a6f6f162b280f9322e41e4878d3d1a351df",
  "uuid": "8e32aff9-96f2-4e87-80c7-0db7ab1f8327",
  "version": 4
}
```

the test logic should parse json and assert /kdf/function property is "scrypt"